.travis.yml

```diff
@@ -1,11 +1,69 @@
 sudo: required
+# We need the systemd for the kubeadm and it's default from 16.04+
+dist: xenial
+services:
+- docker
+language: go
+go:
+- "1.12.x"
 
 stages:
   - name: repo-sync
     if: branch = master AND type = push
 
 jobs:
   include:
+    - name: Lint charts
+      stage: lint
+      env:
+        - CT_VERSION=2.3.3
+        - HELM_VERSION=v2.14.0
+      before_script:
+        - sudo pip install yamale yamllint
+        - curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+        - chmod 700 get_helm.sh
+        - DESIRED_VERSION=$HELM_VERSION ./get_helm.sh
+        - curl -Lo chart-testing_${CT_VERSION}_linux_amd64.tar.gz https://github.com/helm/chart-testing/releases/download/v${CT_VERSION}/chart-testing_${CT_VERSION}_linux_amd64.tar.gz
+        - tar xzf chart-testing_${CT_VERSION}_linux_amd64.tar.gz
+        - sudo chmod +x ct && sudo mv ct /usr/local/bin/
+        - sudo mv etc /etc/ct
+        - rm -f chart-testing_${CT_VERSION}_linux_amd64.tar.gz
+      script:
+        - cd .. # ct is having issues when processing charts on the root
+        - ct lint --all --config charts/ct.yaml # --all is being used as our project structure doesn't allow propper git change detection
+    - name: Tests
+      stage: Tests
+      env:
+        - CHANGE_MINIKUBE_NONE_USER=true
```

Comment:
it would be nice to comment the need for this line

Reply:
done